### PR TITLE
Create TheIdServer.yaml

### PR DESCRIPTION
**Pull request by aguacongas**
# TheIdServer

[OpenID/Connect](https://openid.net/connect/), [OAuth2](https://oauth.net/2/) and [WS-Federation](https://docs.oasis-open.org/wsfed/federation/v1.2/os/ws-federation-1.2-spec-os.html) server based on [IdentityServer4](https://identityserver4.readthedocs.io/en/latest/) or [Duende IdentityServer](https://duendesoftware.com/products/identityserver) needs your help.

### CODE REVIEW

File: _data/projects/TheIdServer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: TheIdServer
+desc: OpenID/Connect, OAuth2 and WS-Federation server based on IdentityServer4 or Duende IdentityServer with its admin UI
+site: https://github.com/Aguafrommars/TheIdServer#readme
```
Comment (ritwik12) [inline _data/projects/TheIdServer.yaml]: @aguacongas Please make it https://github.com/Aguafrommars/TheIdServer

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/TheIdServer.yaml (hunk 1 above)
Comment (aguacongas) [reply _data/projects/TheIdServer.yaml]: Done

**Comment by aguacongas** [comment]
@ritwik12 change done

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@aguacongas Thanks!!

**Pull request merged**